examples/gnrc_border_router/Makefile

```diff
@@ -0,0 +1,48 @@
+# name of your application
+APPLICATION = gnrc_border_router
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= samr21-xpro
```

Comment:
I changed this to samr21-xpro since it is the board we tested on and since native has no support for `periph/uart`.